src/utils/loaders/getCityFeed.js

```diff
@@ -1,7 +1,4 @@
-function add(a, b) {
-    return a + b
+export async function getCityFeed(token) {
+  const response = await fetch(`https://api.waqi.info/feed/here/?token=${token}`);
```

Comment:
url from the ENV file, please. 

Please note: we are gonna use Axios, we are gonna replace Axios next sprint

Reply:
updated code and test to use axios

the whole URL is now from ENV file, not jus the token part (that was already from ENV file)